views/templates/hook/payment_klarna.tpl

```diff
@@ -27,6 +27,17 @@
                     <option value="2">{l s='She/her' mod='buckaroo3'}</option>
                 </select>
             </div>
+            <div {if (!is_null($houseNumber))} style="display: none" {/if}>
+                <div class="col-xs-5">
+                    <label class="required">{l s='House number' mod='buckaroo3'}:</label>
+                </div>
+                <div class="col-xs-7">
+                    <input name="bpe_klarna_house_number" id="bpe_klarna_house_number"
+                           value="{$houseNumber|escape:'html':'UTF-8'}"
+                           type="text"
+                           class="form-control bk-form-control-large"/>
+                </div>
+            </div>
```

Comment:
why do you need to add a form field, why not just add a validation error "Invalid shipping address, a house number is required" ?

Reply:
I cant add a street number field in the address form, the way I get it is by splitting the address and getting the number from there if I just throw an error here it is not clear where you are supposed to add the house number in the address.
This way if you set it correctly it will not show but if you didnt you have to add it